src/strangerThings.js

```diff
@@ -53,7 +64,10 @@ async function setCSS(character) {
 
 async function illuminateLight(index) {
 	const message = getCharacters();
-
+	for(let i=0; i<message.length;i++)
```

Comment:
Use `forEach` method here on the array.

Reply:
Bhaiya I tried this:
message.forEach(element =>await setCSS(element) );

But it is not working. It's saying that await expressions are only allowed within async functions.
so if i tried this:
message.forEach(async function illumine(element) {
		await setCSS(element)
	});
It is saying that jest is not completed after 1 second

Reply:
Ahh okay. This is related to promises. You can leave this for now, or read about them online if you want to.